robot/rover/RoverSetup.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+# Script which setups the Space Concordia Robotics Software team's OBC.
+
+REPO="/home/$USER/Programming/robotics-prototype"
+
+APPEND_TO_BASH="
+
+export ROS_MASTER_URI=http://localhost:11311
+export ROS_HOSTNAME=$USER
+
+. $REPO/robot/rospackages/devel/setup.bash
+. $REPO/venv/bin/activate
+source $REPO/robot/basestation/config/.bash_aliases
+
+"
+
+FINAL_MESSAGE="The script will now exit, ensure that ros is install correctly and that
+all of the rospackages are present. Reboot to run systemd services."
+
+## START
+
+#check if in proper directory
+SCRIPT_DIRECTORY=$REPO/EnvironmentSetup.sh
+if [ ! -f $SCRIPT_DIRECTORY ]
+then
+   # -e enables text editing, \e[#m sets a text colour or background colour. \e[0 ends the edit.
+   echo -e "\e[31m\e[47mYou did not setup the repo directory correctly. Refer to README\e[0m"
+   exit 1
+fi
+
+
+#install prereqs
+sudo add-apt-repository ppa:deadsnakes/ppa -y
+sudo apt update -y
+sudo apt install python3.6 python3.6-venv nodejs npm openssh-server -y
+sudo ufw allow ssh
+
+# Setup venv
+cd $REPO
+python3.6 -m venv venv
+source venv/bin/activate
+
+# Install Requirements
+pip install -U pip
+pip install pyserial==3.4 rospkg==1.1.7
```

Comment:
since these are included in `requirements.txt` perhaps we should just use that? i.e. `pip install -r requirements.txt -r requirements-dev.txt` for completeness